.github/workflows/ci.yml

```diff
@@ -15,25 +15,52 @@ jobs:
     timeout-minutes: 20
 
     outputs:
-      image: ${{ steps.tag.outputs.image }}
+      tag: ${{ steps.setup.outputs.tag }}
 
     steps:
-      - uses: actions/checkout@v2
-
-      - name: build image
-        uses: docker/build-push-action@v1
+      -
+        uses: actions/checkout@v2
```

Comment:
Maybe, for the consistency, use this style everywhere.

Reply:
sure